bucket/quassel.json

```diff
@@ -0,0 +1,49 @@
+{
+    "description": "Modern, cross-platform, distributed IRC client based on the Qt framework.",
+    "homepage": "https://quassel-irc.org",
+    "license": "GPL-2.0|GPL-3.0",
+    "version": "0.14.0",
+    "url": "https://quassel-irc.org/pub/quassel-windows-0.14.0.7z",
```

Comment:
You can check the app is 32 or 64-bit using pelook (`scoop install pelook`)
If the app is 32-bit, then keep as it is (both 32-bit and 64-bit machine can run it)
If the app is 64-bit. then use
```
"architecture" {
    "64bit":  {
        "url": ...
        "hash": ...
    }
```